src/com/hms/flexyosisoftconnector/Main.java

```diff
@@ -109,10 +110,12 @@ private static void getNewPayloadToSend() {
       }
     } catch (IOException e) {
       Logger.LOG_EXCEPTION(e);
+      Logger.LOG_WARN("Exception thrown when obtaining new batch of datapoints. Will retry.");
     }
 
     // if there are data points to send
     if (queuePoints.size() != 0) {
+      Logger.LOG_DEBUG("Grabbed " + queuePoints.size() + " datapoints for proccessing.");

```

Comment:
inconsistent datapoint reference

Reply:
i've been trying to use datapoint everywhere and keep slipping up on that one

Reply:
fixed